nautobot/ipam/models.py

```diff
@@ -991,13 +991,9 @@ def clean(self):
         self.dns_name = self.dns_name.lower()
 
     def save(self, *args, **kwargs):
-        if not self.present_in_database:
-            if self._namespace is None:
-                raise ValidationError({"parent": "Namespace could not be determined."})
-            namespace = self._namespace
-        else:
-            namespace = self.parent.namespace
```

Comment:
This line is preventing `Namespace` from being changed to the intended `Namespace` in the `IPAddressEditForm`